dapr/proto/runtime/v1/appcallback.proto

```diff
@@ -82,6 +82,20 @@ message TopicEventRequest {
   // The matching path from TopicSubscription/routes (if specified) for this event.
   // This value is used by OnTopicEvent to "switch" inside the handler.
   string path = 9;
+
+  // The cloud event extension attributes (custom attributes).
+  // Compatible with the cloud events v1.0.2 spec.
+  map<string, CloudEventAttributeValue> attributes = 10;
+
+}
+
+message CloudEventAttributeValue {
```

Comment:
This is a subset of the options from the official CloudEvent proto spec. They define additional string fields for URLs, but we cannot distinguish between these types because we are not receiving data via the cloudevents proto. Similarly for the Time field defined in that spec.